resources/js/components/LabelInput.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Grid, TextField } from '@mui/material'
+
+interface Props {
+  label: string
+  labelValue: string
+}
+
+const LabelInput: React.VFC<Props> = (props) => {
+  return (
+    <Grid
+      item
+      sx={{ display: 'flex', justifyContent: 'space-between' }}
+      xs={12}
+      mt={3}
+      lg={3}
```

Comment:
```suggestion
      mt={4}
      lg={4}
```
@masato5579 PCだとだいぶ窮屈にみえます。  
特にラベル名が全角3文字で見切れるのは実用的ではない気がします。
3つずつ並べる/文字を小さくする/要素の間の余白を大きくする などの工夫が必要かと。